certif/app/components/auth/register-form.js

```diff
@@ -53,6 +54,18 @@ export default class RegisterForm extends Component {
     try {
       await userRecord.save();
 
+      certificationCenterInvitationResponseRecord = this.store.createRecord(
+        'certification-center-invitation-response',
+        {
+          id: this.args.certificationCenterInvitationId,
+          code: this.args.certificationCenterInvitationCode,
+          email: this.email,
+        }
+      );
+      await certificationCenterInvitationResponseRecord.save({
+        adapterOptions: { certificationCenterInvitationId: this.args.certificationCenterInvitationId },
+      });
+
       this.password = null;
```

Comment:
Chipotage : ajouter un test sur le this.password= null; 

ex 
`assert.strictEqual(component.password, null);`

Reply:
Si ce bout de code est important ^^

Reply:
En soit ce bout de code pourrait être retiré parce qu'il n'a pas de sens. Vu que l'on connecte l'utilisateur donc il se redirigé vers une nouvelle page.